COORDINATION.md

```diff
@@ -66,6 +68,21 @@ The virtio-blk device presents its driver in an option ROM (OROM) for UEFI / BIO
 
 The NVMe device driver will poll the CSTS.rdy bit to ensure that infrastructure backend is ready before reading or writing.
 
+### idpf
+
```

Comment:
please consider adding to terms table what IDPF is... I guess `Infrastructure Data-Plane Function` ?

Reply:
added https://github.com/opiproject/opi-prov-life/commit/3846f52fb8fd8e779929541b8e3350bb160165e3